odk1-src/form-question-types.rst

```diff
@@ -1691,6 +1699,29 @@ Records video, using the device camera.
   video, blinking, Please record a video of yourself blinking., Three times is probably sufficient.
 
 
+.. _selfie-video-widget:
```

Comment:
Let's match the other selfie widget and call this `self-portrait-video-widget` and rename the other one `self-portrait-image-widget`

Reply:
Done.